### PR TITLE
Device info draft

### PR DESCRIPTION
**Pull request by thundron**

### CODE REVIEW

File: lib/gui/app/app.ts [hunk 1]
```diff
@@ -217,8 +217,7 @@ function prepareDrive(drive: Drive) {
 			disabled: true,
 			icon: 'warning',
 			size: null,
-			link:
-				'https://www.raspberrypi.org/documentation/hardware/computemodule/cm-emmc-flashing.md',
+			link: 'https://www.raspberrypi.org/documentation/hardware/computemodule/cm-emmc-flashing.md',
```
Comment (lurch) [inline lib/gui/app/app.ts:220]: This documentation is now at https://www.raspberrypi.com/documentation/computers/compute-module.html#flashing-the-compute-module-emmc

Comment (thundron) [reply lib/gui/app/app.ts:220]: thanks for the heads up! I'll make sure to include it next CI run, as even a simple doc change for now triggers the whole process all over again :/

**Comment by thundron** [comment]
@balena-ci I self-certify!

File: lib/gui/app/app.ts (hunk 1 above)
Comment (thundron) [reply lib/gui/app/app.ts:220]: including in the latest PR!

**Pull request merged**